articles/dashboard/index.md

```diff
@@ -15,14 +15,18 @@ classes: topic-page
 <ul class="topic-links">
   <li>
     <i class="icon icon-budicon-715"></i><a href="/dashboard/creating-users-in-the-management-portal">Create Users in the Dashboard</a>
+    <p>How to create end users using the Dashboard</p>
```

Comment:
@khorne3 For future reference, the `p` element should be within the `li`. See the diff below.

Nested within:

![image](https://user-images.githubusercontent.com/11715799/32896153-f5a3f19e-caea-11e7-9d84-b5f40a4c52a7.png)

When put after the li:
![image](https://user-images.githubusercontent.com/11715799/32896180-05f847ca-caeb-11e7-95a3-bbfda92ce2cf.png)